is08/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2018 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from is08.action import Action
```

Comment:
Do these need to be here or can you get away with an empty __init__ and rely upon its default behaviour? Given that other imports do 'from is08.action import Action' rather than 'from is08 import Action' it may not be required at all? Or have I misunderstood?

Reply:
Yeah, think I started doing it a different way then changed it and forgot those were in there :)